app/(main)/(routes)/documents/[documentId]/page.tsx

```
@@ -1,45 +1,58 @@
+  const { documentId } = useParams();
+  const Editor = useMemo(
+    () => dynamic(() => import("@/components/editor"), { ssr: false }),
+    [],
+  );
 
-interface DocumentIdPageProps {
-    params: {
-        documentId: Id<"documents">
-    };
-}
+  // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)